site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -104,7 +104,7 @@ If you do not intend to use the Login API, or will only be calling this API from
 The name of the Application.
 
 [field]#application.oauthConfiguration.authorizedOriginURLs# [type]#[Array<String>]# [optional]#Optional#::
-An array of URLs that are the authorized origins for FusionAuth OAuth.
+An array of URLs that are the authorized origins for FusionAuth OAuth. If configured, these URLs will be allowed to embed a FusionAuth hosted login page, otherwise the `X-Frame-Options` header will be set to `DENY`.
```

Comment:
Suggestion: 

>
> One or more HTTP origins that are allowed to use the browser based grants. This configuration is optional, and if omitted, all HTTP origins will be allowed. 
> {empty line}
>For security purposes, all FusionAuth hosted login pages will add an HTTP response header of `X-Frame-Options: DENY`. This response header will restrict the page from being loaded in an Iframe.  If you intend to utilize an Iframe to load one or more FusionAuth hosted login pages, you must configure your HTTP Origin so that FusionAuth will remove this HTTP response header and allow the page to load.

We may also want to add a short comment on support of this strategy. It should work, but it is not a tested use case that we document anywhere else.